lib/puppet/provider/gnupg_key/gnupg.rb

```diff
@@ -73,9 +76,9 @@ def add_key_from_key_content
     path = create_temporary_file(user_id, resource[:key_content])
     command = "gpg --import #{path}"
     begin
-      output = Puppet::Util::Execution.execute(command, :uid => user_id, :failonfail => true)
+    output = Puppet::Util::Execution.execute(command, :uid => user_id, :failonfail => true)
     rescue Puppet::ExecutionFailure => e
-      raise Puppet::Error, "Error while importing key #{resource[:key_id]} using key content:\n#{output}}"
+      raise Puppet::Error, "Error while importing key #{resource[:key_id]} using key content: #{output}"
```

Comment:
I actually want the output of the command in case of failure, and that explains why I was getting nothing when trying to debug it earlier. what do Sean what do?